plugins/callback/print_task.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Max Mitschke <maxmitschke@fastmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: print_task
+type: aggregate
+short_description: Prints playbook task snippet to job output
+description:
+  - This is an Ansible callback plugin that will print the currently executing
+    playbook task to the job output.
```

Comment:
It's clear from the documentation that this is an Ansible callback plugin :)
```suggestion
  - This plugin prints the currently executing playbook task to the job output.
```